sample/solidus_sample.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
 
   s.author      = 'Solidus Team'
   s.email       = 'contact@solidus.io'
-  s.homepage    = 'http://solidus.io'
+  s.homepage    = 'https://solidus.io'
```

Comment:
Good catch! Better on another commit, too, as it has nothing to do with the description.